CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "armv7l")
 	set(CMAKE_C_FLAGS "-Ofast -fPIC -mfloat-abi=hard -mfpu=vfp -march=armv7-a -ffast-math -Wfloat-equal -Wshadow -Wcast-align -Wstrict-overflow=5 -Wwrite-strings -Waggregate-return -Wswitch-default -Wswitch-enum -Wformat=2 -g -Wall -Wno-sign-conversion -Wunreachable-code -Wstrict-prototypes")
 endif()
 
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "^mips")
+	set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE armhf)
```

Comment:
You want to set `CPACK_DEBIAN_PACKAGE_ARCHITECTURE` to `mipsel` (that's the only Debian arch which will run on the CI20 anyway). `CMAKE_SYSTEM_PROCESSOR` is set to the output of `uname -m` which doesn't distinguish between big and little endian so you can just leave the if check as it is.